docker/development/build.sh

```diff
@@ -1,7 +1,23 @@
 #!/usr/bin/env bash
 
-# This will be ran with the root directory as the docker context
+# This builds the sandtank-dev docker image from the git repo pull in the current directory
+# This is the first base image of the sandtank application. This is used by the runtime docker image build.
```

Comment:
I'm not sure I understand what you mean by `first` and `second` base images. Can you explain this a bit? Thank